facade/src/main/java/org/jboss/pnc/facade/impl/BrewPusherImpl.java

```diff
@@ -272,67 +203,109 @@ private BuildRecord getLatestSuccessfullyExecutedBuildRecord(Base32LongID buildR
     }
 
     @Override
-    public boolean brewPushCancel(String buildId) {
+    public void cancelBuild(String buildId) {
```

Comment:
Don't you want to use name like `cancelBuildPush`?

Reply:
it's BrewPusher class so it's all about brew push. But I guess I can rename it